CONTRIBUTING.md

```diff
@@ -322,3 +324,22 @@ precompile/build-all-addons.js
 ```
 
 NOTE: It is recommended to comment out the ABI versions you don't want to generate/regenerate in the `build-all-addons` script. Alternatively, if you want to regenerate all builds that's fine too, but it takes longer.
+
+#### How to publish local layer to AWS
+
+To publish local layer to AWS with the AWS CLI, the CLI needs to have access to the IAM credentials.
+
+Steps to follow
+
+- Create an IAM user if you don't already have one with the required rights.
+- Create `credentials` file on the following folder location, `/usr/local/aws`
+- Add below informations to the newly created `credentials` file. The access key and secret key can be obtained from the IAM user account
+```javascript
+  - [default]
+  - aws_access_key_id = <add your access key>
+  - aws_secret_access_key = <add your secert key>
+```
+    More information can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
+- Run these commands to publish local layer to AWS
+  - cd packages/aws-lambda/layer/bin/
+  - REGIONS=<region> SKIP_DOCKER_IMAGE=true BUILD_LAYER_WITH=local LAYER_NAME=<aws-layer-name> ./publish-layer.sh
```

Comment:
Suggestion: Either add a comment to explain that we should use "experimental-instana-nodejs-with-extension" or set the value here directly :)